benchmarking/platforms/platform_base.py

```diff
@@ -78,9 +78,12 @@ def preprocess(self, *args, **kwargs):
     def copyFilesToPlatform(self, files, target_dir=None):
         target_dir = (self.tgt_dir if target_dir is None else target_dir)
         if isinstance(files, string_types):
-            target_file = os.path.join(target_dir, os.path.basename(files))
-            self.util.push(files, target_file)
-            return target_file
+            if not os.path.isdir(files):
+                target_file = os.path.join(target_dir, os.path.basename(files))
+                self.util.push(files, target_file)
+                return target_file
+            else:
+                return files
```

Comment:
This doesn't work with ios/android. It's better to change the push routine to support directory there.

https://github.com/facebook/FAI-PEP/blob/master/benchmarking/platforms/host/hdb.py#L23